com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -652,9 +666,11 @@ half4 UniversalFragmentPBR(InputData inputData, half3 albedo, half metallic, hal
     return half4(color, alpha);
 }
 
-half4 UniversalFragmentBlinnPhong(InputData inputData, half3 diffuse, half4 specularGloss, half smoothness, half3 emission, half alpha)
+half4 UniversalFragmentBlinnPhong(InputData inputData, half3 diffuse, half4 specularGloss, half smoothness, half3 emission, half alpha, half occlusion = 1.0)
 {
     Light mainLight = GetMainLight(inputData.shadowCoord);
+
+    inputData.bakedGI *= occlusion;
```

Comment:
why not apply occlusion to bakedGI before calling this function?

Reply:
In the latest commit I've now added direct and indirect AO values so I'd like to have the AO sampling done here so it's in the same place.